### PR TITLE
MAINTAINERS: Add Bryan Boreham, Ayoub Mrini

### PR DESCRIPTION
**Pull request by bboreham**
Two people who have been quite active in Prometheus recently.

### CODE REVIEW

**Comment by machine424** [comment]
Let’s keep it burning!

File: MAINTAINERS.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ George Krajcsovits (<gyorgy.krajcsovits@grafana.com> / @krajorama)
     * `module`: Augustin Husson (<husson.augustin@gmail.com> @nexucis)
 * `Makefile` and related build configuration: Simon Pasquier (<pasquier.simon@gmail.com> / @simonpasquier), Ben Kochie (<superq@gmail.com> / @SuperQ)
 
+Bryan Boreham (<bjboreham@gmail.com> / @bboreham) and Ayoub Mrini (<ayoubmrini424@gmail.com> / @machine424) are general maintainers.
```
Comment (beorn7) [inline MAINTAINERS.md:22]: Maybe merge this paragraph with the top one (line 3) and give it the form like this:

General maintainers:
* Julien Pivotto (<roidelapluie@prometheus.io> / @roidelapluie)
* Bryan Boreham (<bjboreham@gmail.com> / @bboreham)
* Levi Harrison (<levi@leviharrison.dev> / @LeviHarrison)
* Ayoub Mrini (<ayoubmrini424@gmail.com> / @machine424)

Some parts of the codebase have other maintainers:
...

**Review by beorn7 — APPROVED** [review]
Approving the "promotion" in general, with a suggestion how to simplify the structure.

File: MAINTAINERS.md (hunk 1 above)
Comment (bboreham) [reply MAINTAINERS.md:22]: That takes away the "default" concept.  I would say the success of that idea has depended a lot on the availability of Julien and Levi over time.

(And I specifically did not want to add it for myself)

**Review by simonpasquier — APPROVED** [review]

**Review by roidelapluie — APPROVED** [review]

**Pull request merged**